lib/DataFetcher.dart

```diff
@@ -53,7 +76,7 @@ class DataFetcher {
     ];
   }
 
-  static UserData fetchUserData() {
+  static UserData fetchDefaultUserData() {
```

Comment:
Do we actually ever need to fetch fake data like that?

Reply:
For now, I just implemented it like this because if you install the app there will be no saved data so you couldn't test the persistence in a fresh installation.  After the full implementation we can just have a toast message that can't retrieve data and ditch this part.

Reply:
sounds good